CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 * CLI flags --tls-sni-01-port and --tls-sni-01-address have been removed.
 * The values tls-sni and tls-sni-01 for the --preferred-challenges flag are no
   longer accepted.
+* Removed the flags: `--agree-dev-preview` and `--dialog`
+* Removed the flag: `init-script`
```

Comment:
```suggestion
* Removed the flag: `--apache-init-script`
```
This is what the flag actually turns into because it is inside the Apache plugin.

Reply:
my bad, I thought there was a separate command for apache. I'll fix that in the next commit.